app/plugins/eu.esdihumboldt.hale.app.transform/src/eu/esdihumboldt/hale/app/transform/ExecApplication.groovy

```diff
@@ -227,23 +229,18 @@ $baseCommand
 				break
 
 			case '-filter-on':
-			//<Type>:CQL:<EXPRESSION>
-				if(value.findIndexValues {it = ':'}.size() >=2 ){
-					int typeNameIndex = value.indexOf(':')
-					if(typeNameIndex>0){
-						String typeName = value.take(typeNameIndex)
-						String expression = value.substring(typeNameIndex+1)
-						executionContext.filters.addTypeFilter(typeName, expression)
-					}
-				}else {
+				int typeNameIndex = value.indexOf(':')
```

Comment:
I thought user will provide only simple name like "River". He will not supply whole QName as per above. In Hale Studio also, we can see only name, not whole QName.